### DISCUSSION TITLE
SafariDriver unexpectedly exited: Status Code was 1

### DISCUSSION THREAD

**Discussion by Uno-count** [seleniumbase/SeleniumBase#2847] [Q&A] [answered]
Can you help me fix this, I am always having an issue using safaridriver, but the whole test case are running smooth in chromedriver and in firefox, 

![Error in safari](https://github.com/seleniumbase/SeleniumBase/assets/73009596/9466adf0-6c14-4944-9d58-c45030c10644)

**Comment by mdmintz** [comment, marked as answer]
See this thread for SafariDriver: https://forums.developer.apple.com/forums/thread/124461

**Reply by Uno-count** [reply]
@mdmintz Thank you very much, my issue has been fix.


another question is I have this 
@pytest.mark.run(order=1)

but whenever I run it in bulk or in test picking in the sbase gui, the first test in the test file would be always result as fail using safari but in chrome it works perfectly.

**Reply by mdmintz** [reply]
Would need more information to debug that.
